src/lang/modifyAst.ts

```diff
@@ -288,26 +322,34 @@ export function mutateObjExpProp(
 export function extrudeSketch({
   node,
   pathToNode,
-  shouldPipe = false,
```

Comment:
Nice! glad we're getting rid of this

Reply:
it was adding complexity to each of these code-mod functions and not being used.